srcpkgs/nostr-rs-relay/template

```diff
@@ -0,0 +1,17 @@
+# Template file for 'nostr-rs-relay'
+pkgname=nostr-rs-relay
+version=0.8.9
+revision=1
+build_style=cargo
+hostmakedepends="pkg-config protobuf"
+makedepends="openssl-devel"
+short_desc="Nostr relay written in Rust"
```

Comment:
```suggestion
short_desc="Nostr relay"
```

Reply:
I'm fine with all of these. Updated accordingly.